### DISCUSSION TITLE
Grove - Turbidity Sensor Meter for Arduino V1.0 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1163] [Q&A]
# Grove - Turbidity Sensor Meter for Arduino V1.0 | Seeed Studio Wiki

Grove - Turbidity Sensor Meter for Arduino V1.0

https://wiki.seeedstudio.com/Grove-Turbidity-Sensor-Meter-for-Arduino-V1.0/

**Comment by gopalshaha** [comment]
I have recently bought this sensor. Then connected to Arduino and used the given code to take reading. When it was not even inside of any liquid, it showed a reading of around 3.6. Then I put it into clean filtered water, and the value suddenly dropped to 0 and then rose to around 3.5. I also tried to use a little muddy water, and unfortunately, the value only differed a little. It was done several times, but the result was the same in every attempt.

My actual aim is to measure the turbidity in NTU. Is there any other code or library that I can use that will provide accurate measurement of turbidity in the unit NTU? Please suggest.

**Reply by tyleronsi** [reply]
Hey @gopalshaha, I stumbled upon this schematic by Amphenol, the maker of the actual sensor.  It has a calibration curve that converts voltage to NTU.  https://files.seeedstudio.com/products/101020752/Turbidity%20Sensor%20specification.pdf
